src/index.html

```diff
@@ -0,0 +1,84 @@
+---
+layout: default.html
+pageTitle: Home
+---
+
+<section class="homepage-grid">
+    <ul class="u-list-reset" class="homepage-grid-item" id="intro">
+        <li>Second Chance is a</li>
+        <li>501(c)(3) nonprofit that</li>
+        <li>provides people,</li>
+        <li>materials and the</li>
+        <li>environment with a</li>
+        <li>second chance</li>
+    </ul>
```

Comment:
This is not a list (compare the the `ul`s you made on the Job Training page). Instead, if you want to control the line breaks/widths of this text, you should put the entire thing in a `p` element and control it via the CSS width:
```suggestion
    <p class="homepage-grid-item" id="intro">
      Second Chance is a 501(c)(3) nonprofit that provides people, materials and the environment with a second chance
    </p>
```

Using `ch` units for this width can control it by the number of characters, which may be helpful here, for example:

```css
#intro {
  width: 18ch;
}
```